### PR TITLE
Add support for axiom annotations

### PR DESCRIPTION
**Pull request by rctauber**
See #226

### CODE REVIEW

**Comment by jamesaoverton** [comment]
Good. We need:

- to make sure this also works to annotate both annotation axioms and logical axioms
- add/modify at least one unit test to cover those two cases
- update the documentation, keeping our terminology as clear as possible: https://github.com/ontodev/robot/issues/226#issuecomment-357828902

**Comment by rctauber** [comment]
* Logical axiom annotations are supported with `>C`.
* I updated `test/resources/template.csv` to include `>*` template strings. The axiom count stays the same, though, since they're included in the axioms they're annotating. If we wanted to check these specifically, I think we'd have to loop through the axioms and make sure the `rdfs:comment` annotations are included. What are your thoughts on this?
* Documentation updated. I didn't include `>>A` even though it's supported... I think anybody who's new to the technical aspect of ontology development might be overwhelmed by that.

**Comment by jamesaoverton** [comment]
Good. Two things: 

1. I'd feel better if the template was more thoroughly checked. I suggest saving a known-good `template.owl` output of `template.csv` into the repo, and making a unit test against to diff against that.
2. Somebody is going to try `>AL` and wonder why it doesn't work. Can you generalize to handle `AL`, `AT`, `AI`?

File: robot-core/src/main/java/org/obolibrary/robot/TemplateOperation.java [hunk 1]
```diff
@@ -244,7 +241,7 @@ public static boolean validateTemplateString(String template) {
     if (template.equals("CLASS_TYPE")) {
       return true;
     }
-    if (template.matches("^(A|AT|AL|AI|C) .*")) {
+    if (template.matches("^(A|>A|>AT|>AL|>AI|>>A|>>AT|>>AL|>>AI|AT|AL|AI|C|>C) .*")) {
```
Comment (jamesaoverton) [inline robot-core/src/main/java/org/obolibrary/robot/TemplateOperation.java]: Please simplify to something like: `^>?C|^>{0,2}(A|AT|AL|AI)`

**Comment by jamesaoverton** [comment]
I checked this on OBI and MRO, both of which use several templates, and it worked fine.

Please make that small regex simplification and I'll merge.

**Pull request merged**